src/core/gui/toolbarMenubar/model/ColorPalette.cpp

```diff
@@ -79,12 +80,7 @@ auto Palette::parseColorLine(const std::string& line) -> bool {
     return false;
 }
 
-auto Palette::parseCommentLine(const std::string& line) const -> bool {
-    if (line.front() == '#')
-        return true;
-    else
-        return false;
-}
+auto Palette::parseCommentLine(const std::string& line) const -> bool { return line.front() == '#'; }
```

Comment:
Hehe😅, I would call this over redundant. I also wrote such code some time. Especially at 4 o'clock in the morning, this can happen.

PS: I actually searched for a similar word of redundant, which means the same as over descriptive.